app/controllers/hackdays_controller.rb

```diff
@@ -0,0 +1,35 @@
+class HackdaysController < ApplicationController
+  def index
+    @hackdays = Hackday.all
+  end
+
+  def show
+    @hackday = Hackday.find(params[:id])
```

Comment:
use either where or find_by_id.

find raises an exception if not found and you should be catching that in a user displayed error.